sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -50,161 +50,162 @@
 #define TIMEOUT_MQTT_RECEIVE_MS (60 * 1000)
 #define TIMEOUT_MQTT_DISCONNECT_MS (10 * 1000)
 
-static char mqtt_client_id[128];
-static char mqtt_username[128];
-static char register_publish_topic[128];
-static char query_topic[256];
-static char global_provisioning_endpoint[256] = { 0 };
-static char id_scope[16] = { 0 };
-static char registration_id[256] = { 0 };
-static char x509_cert_pem_file[256] = { 0 };
-static char x509_trust_pem_file[256] = { 0 };
+// Logging
+#define LOG_ERROR(...) \
```

Comment:
I think if we do this, these should probably go into some common `.c` or `.h`. This seems like something else to be copied and pasted across samples and then possibly get out of sync.

Reply:
Totally.